docs/Contributing.md

```diff
@@ -25,44 +25,43 @@ To contribute, please fork the repo and develop in a feature branch off the curr
 ### For non-Windows ###
 1. Run `build.sh`
 
-### Changing build configurations ###
-To build in `DEBUG` mode - set the environment variable `DN3B` to `DEBUG`
-To build in `RELEASE` mode (default) - set the environment variable `DN3B` to `RELEASE`
-
 ## Running ##
 1. Run `dotnet new3` at the command line
 
 ### Note for Windows users ###
 The location `dotnet-new3.exe` gets built to will be placed at the start of the `PATH` environment variable, so it won't become available in console windows (other than the one you've built in) that are already open. To run in already open windows, you can add the `dev` directory (created during the build) to the `PATH` environment variable, or run `dotnet-new3.exe` from that directory.
 
-### Note for non-Windows users ###
-`setup.sh` attempts to create a symlink to the `dotnet-new3` executable in `/usr/local/bin/`, if the attempt to elevate to do that is denied, you can still run the `dotnet-new3` executable directly in the `dev` directory created during the build.
-
 [Top](#top)
 
 # Coding Style #
 
-* All block statements _must_ use braces (if/else, for, foreach, using, etc.)
-  * Immediately nested usings do not need braces at the outer layers
-* We use four spaces of indentation (no tabs).
+Most of the styling is enforced by analyzers and the rules covered by the analyzers are not listed in this section. Therefore, it is highly recommended to use an IDE with Roslyn analyzers support (such as Visual Studio or Visual Studio Code).
+
 * We use explicit types (no usages of `var`)
```

Comment:
Isn't this changing with switch to .NET Runtime rules?

Reply:
There are currently no rules to enforce this, so I left it here but I didn't realize that it differed from runtime. I will update the text to align with the runtime rules.